### ISSUE TITLE
Move to maven.scijava.org

### ISSUE THREAD

**Issue by tischi** [tischi/imagej-utils#13]
@imagejan 
I tried to update to version `9.0.5` of bigdataviewer-core, but got:

```
Exception in thread "main" java.lang.NoClassDefFoundError: mpicbg/spim/data/sequence/DefaultVoxelDimensions
```

Then I fixed it by adding the `spim_data` dependency, but got other errors. 
I think I need to go to a more recent version of scijava parent pom, but which one? 
Currently I am on `27.0.1`.

**Comment by tischi** [comment]
@imagejan 
1. I have a branch now where I think it could be worth trying the CI:
https://github.com/tischi/imagej-utils/tree/scijava29.1.0
    - Does the work for testing or do I have to push to master?
2. In fact, a collaborator would like to depend on this repo so it would be nice to upload it to maven.scijava.org

**Comment by imagejan** [comment]
@tischi the build fails for me because `image-transform-converters-0.1.1` cannot be found on the imagej maven repository. We started deployment with `image-transform-converters-0.1.2-SNAPSHOT` and there's no release version yet. Would you mind running [`release-version.sh`](https://github.com/scijava/scijava-scripts/blob/6b25a0970cbc5eed9df7a4822f1dbc04a92ee0ac/release-version.sh) on that repository?

**Comment by tischi** [comment]
@imagejan I tried to release a version 0.1.2. Did it work?

**Comment by imagejan** [comment]
@tischi 🎉 your `0.1.3` finally worked!

https://maven.scijava.org/#nexus-search;gav~org.itc~image-transform-converters~0.1.3~~

**Comment by tischi** [comment]
Cool! So this repo is next?
How could I find JavaDoc errors beforehand?

**Comment by imagejan** [comment]
Yes, see https://github.com/tischi/imagej-utils/pull/15 for an intermediate pull request.

> How could I find JavaDoc errors beforehand?

As [mentioned on Gitter](https://gitter.im/scijava/scijava-common?at=5f2c15f645cf645e4a11f075) already, you can use `mvn javadoc:javadoc` locally.

**Comment by tischi** [comment]
@imagejan 
I just merged our development branch into master.
Could you go on with setting up travis?

**Comment by imagejan** [comment]
I force-pushed #16, it is ready to be reviewed and merged now.

**Comment by imagejan** [comment]
🎉 first `SNAPSHOT` deployed to maven.scijava.org: https://maven.scijava.org/#nexus-search;gav~de.embl.cba~imagej-utils~~~

Run `/scijava/scijava-scripts/release-version.sh` on your `master` branch (after making sure you're up-to-date with `origin`) to cut a new release anytime.

**Comment by tischi** [comment]
Awesome! It worked straight away!
Thank you sooooo much!!!

**Comment by imagejan** [comment]
@tischi wrote:
> It worked straight away!

Hm, looks like it actually *didn't*:

https://travis-ci.com/github/tischi/imagej-utils/builds/179096635

but I have no idea why... `[ERROR] The git-clone command failed.` could you try to restart the build on travis (there should be a button on the top right for you on that webpage).

**Comment by tischi** [comment]
Where should I click?

![image](https://user-images.githubusercontent.com/2157566/89783203-d5be3f80-db16-11ea-9040-5244e9b15a8a.png)

**Comment by imagejan** [comment]
@tischi you probably have to sign in to Travis first (via your GitHub account), then you should be able to restart the build in the *More options* menu, or with a button just below that menu.

Here's how it looks for me (on a project where I have write access):

<img width="221" alt="image" src="https://user-images.githubusercontent.com/2033938/89783512-58df9580-db17-11ea-8c13-f9a0430e48e2.png">

**Comment by tischi** [comment]
ok, I did it and it still says `#8 failed` at the top, and `exited 1` at the bottom, but it does not give any error message as far as I can see...

**Comment by imagejan** [comment]
@tischi you can expand the error messages on Travis by clicking on the little triangles next to the line numbers ;-)

The `<developerConnection>` URL contained an error, see #17 for a quick fix.

**Comment by tischi** [comment]
nice! now it worked I think!
